pkg/vulnsrc/osv/osv_test.go

```diff
@@ -108,3 +109,76 @@ func TestVulnSrc_Update(t *testing.T) {
 		})
 	}
 }
+
+func TestVulnSrc_PythonGHSAIDsIsSkipped(t *testing.T) {
+	tests := []struct {
+		name          string
+		dir           string
+		pythonBuckets []wantKV
+		rustBuckets   []wantKV
+	}{
+		{
+			name: "happy path, when python GHSA-ID is skipped",
```

Comment:
I think we can merge this case to TestVulnSrc_Update.

Reply:
Done